addons/website_slides_survey/tests/test_course_certification_stats.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo.addons.survey.tests.common import TestSurveyCommon
+
+
+class TestCourseCertificationStats(TestSurveyCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # Create certifications
+        cls.certifications = cls.env["survey.survey"].create(
+            [
+                {
+                    "title": f"certification {i + 1}",
+                    "scoring_type": "scoring_with_answers",
+                    "certification": True,
+                }
+                for i in range(3)
+            ]
+        )
```

Comment:
Can we shorten those using this syntax:

```py
cls.certifications = cls.env["survey.survey"].create([{
    "title": f"certification {i + 1}",
    "scoring_type": "scoring_with_answers",
    "certification": True,
} for i in range(3)])
```

As said earlier there is a lot of boiler plate data so let's avoid empty lines 🙂 

(To apply to other ones when possible).